### ISSUE TITLE
Oxigraph server Compatibility with `SPARQLWrapper`

### ISSUE THREAD

**Issue by kuzeko** [oxigraph/oxigraph#87]
I've setup a entrypoint with the webserver and tried to connect via `SPARQLWrapper` as follows:

```python

from SPARQLWrapper import SPARQLWrapper, JSON
endpoint = 'http://localhost:7878/query'
sparql = SPARQLWrapper(endpoint)
sparql.setTimeout(300)
#sparql.setReturnFormat(JSON)
sparql.setQuery('SELECT * WHERE { ?s ?p ?o} LIMIT 2')
results = sparql.query()
print(results)
```

But the server returns an error:

```
QueryBadFormed: QueryBadFormed: a bad request has been sent to the endpoint, probably the sparql query is bad formed. 

Response:
b'Unexpected parameter: format'
```

**Comment by Tpt** [comment]
Thank you for the feedback!

Oxigraph server is currently quite pedantic about the SPARQL request and returns an error when unknown query parameters are passed to the system. I just pushed e92c36f115bd536ec00973b8dcb7aadd3690db72 that allows unknown query parameters.

**Comment by kuzeko** [comment]
Nice, thanks! Once this is deployed in the docker I will test again.

**Comment by Tpt** [comment]
I just released v0.2.2. The new image is in Docker hub.

Feel free to open the issue again if it does not work properly.

**Comment by kuzeko** [comment]
Seems to work!  Thanks a lot
